modules/ensemble/lib/action/Log_event_action.dart

```diff
@@ -7,31 +7,47 @@ import 'package:ensemble/util/utils.dart';
 import 'package:ensemble_ts_interpreter/invokables/invokable.dart';
 import 'package:flutter/cupertino.dart';
 
-class LogEvent extends EnsembleAction {
-  final String eventName;
+class TrackEvent extends EnsembleAction {
```

Comment:
why change the name? it is Logevent as you are logging events to the server that may or may not be used for tracking, please change it back to LogEvent.

Reply:
that's ok, this is generic right, other tools may call it something else. TrackEvent just doesn't make sense.